irohad/ametsuchi/block_query.hpp

```diff
@@ -43,57 +40,64 @@ namespace iroha {
        * @param account_id - account_id (accountName@domainName)
        * @return observable of Model Transaction
        */
-      virtual rxcpp::observable<model::Transaction> getAccountTransactions(
-          const std::string &account_id) = 0;
+      virtual rxcpp::observable<
```

Comment:
Maybe it is better to declare some protected alias?
Like:
`using wTransaction = std::shared_ptr<shared_model::interface::Transaction>`